scripts/clean-test-data.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# A script to download and untar test data
+#
+
+# Change directory to the `test` folder.
+cd "$(dirname "$0")/../test"
+
+# Clean saco dataset.
+rm -rf soco
```

Comment:
`soco` isn't special; we could put that into `data` (and have the tests use that spelling) & then everything would be in `data`, one-stop shopping